src/Core/Enums/PolicyType.cs

```diff
@@ -6,5 +6,6 @@ public enum PolicyType : byte
         MasterPassword = 1,
         PasswordGenerator = 2,
         OnlyOrg = 3,
+        RequireSso = 4
```

Comment:
Formatting: Please trail the ending of enum lists, object property initializers, etc. with a comma, so just like the `OnlyOrg` above didn't have to be modified to add the comma, when policy type `5` get's added the dev won't have to modify this line here to add the comma.

Reply:
Will update!

Reply:
Updated.